ruma-events/src/key/verification/accept.rs

```diff
@@ -1,11 +1,13 @@
 //! Types for the *m.key.verification.accept* event.
 
+use std::collections::BTreeMap;
+
 use ruma_events_macros::BasicEventContent;
 use serde::{Deserialize, Serialize};
+use serde_json::Value;
```

Comment:
Please rename like everywher else.

```suggestion
use serde_json::Value as JsonValue;
```